webui/card_partial.html

```diff
@@ -36,12 +36,8 @@
     <div class="spread_view rel" style="background-image:url({{ image_host }}/spread/{{ card.id }}.png)">
       <a class="spread_link" style="height:100%;width:100%;display:block" target="_blank" href="{{ image_host }}/spread/{{ card.id }}.png"></a>
         {% if card.has_sign %}
-          {% if not request.is_mobile %}
           <a class="sign_view" target="_blank" href="{{ image_host }}/sign/{{ card.id }}.png">
-          {% else %}
-          <a class="sign_view_m" target="_blank" href="{{ image_host }}/sign/{{ card.id }}.png">
-          {% end %}
-          <img src="{{ image_host }}/sign/{{ card.id }}.png" style="width:50%" alt="{{ _("SSR Signature") }}" />
+          <img src="{{ image_host }}/sign/{{ card.id }}.png" style="width:35%" alt="{{ _("SSR Signature") }}" />
```

Comment:
I'm removing tag `a` because it won't rezise dynamically and eventually blank area also covers `spread_link`. For now you can save img by right click instead of providing same experience as `spread_link `did.